src/backend/config.ts

```diff
@@ -621,8 +621,8 @@ class GlobalConfigV0 extends GlobalConfig {
         name: userName
       },
       wineCrossoverBottle: 'Heroic',
-      winePrefix: isWindows ? defaultWine : `${userHome}/.wine`,
-      wineVersion: isWindows ? {} : defaultWine
+      winePrefix: isWindows ? {} : join(heroicDefaultWinePrefix, 'default'),
```

Comment:
Is this empty object for windows intended? It could lead to unexpected behavior, since we expect winePrefix as a string

Reply:
You are right. Setting a empty string makes more sense